SpriterDotNet/SpriterParser.cs

```diff
@@ -13,6 +13,7 @@ namespace SpriterDotNet
     public static class SpriterParser
     {
         public static readonly ICollection<ISpriterParser> Parsers = new List<ISpriterParser>();
+        internal static Action<Spriter> InitAction = parser => { Init(parser); };
```

Comment:
Main problem is that there's no way to test that Parse calls Init when successful. By exposing it like this you can test that the one method calls the other, and thereby get full code coverage. You could test that parse returns a Spriter with the pivots set correctly, but then you aren't testing at the unit level.